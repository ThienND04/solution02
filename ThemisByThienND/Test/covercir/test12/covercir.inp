5
20694015920305806 8005040426152053  3184854504917518
1955479025838375 1775729312163293  1277928256277633
3773371627617695 56519543057614  912665813373214
255663087950267168 18688170576089770  116285685026591596
63126284706937305 7135279441800268  2092662885688670
