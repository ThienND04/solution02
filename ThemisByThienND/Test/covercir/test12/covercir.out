12189281973125692
1137718998597029
2230198994973263
137909553816677503
34084450631524450
